sdk/dotnet/Pulumi/Serialization/Attributes.cs

```diff
@@ -76,4 +76,12 @@ public sealed class OutputTypeAttribute : Attribute
     public sealed class OutputConstructorAttribute : Attribute
     {
     }
+
+    /// <summary>
+    /// Attribute used by a Pulumi Cloud Provider Package to mark enum types.
```

Comment:
Should we copy your nice comment about the expected structure of the enum type in here?